### ISSUE TITLE
qt6ct not detecting installed qt styles

### ISSUE THREAD

**Issue by zspher** [NixOS/nixpkgs#350514]
### Describe the bug

qt6ct does not show the option of installed qt styles

### Steps To Reproduce

1.

```nix
  environment.systemPackages = with pkgs; [
    kdePackages.breeze
    kdePackages.qtstyleplugin-kvantum
  ];
```

or in home-manager

```nix
  qt = {
    enable = true;
    style.package = with pkgs; [
      kdePackages.breeze
      kdePackages.qtstyleplugin-kvantum
    ];
    platformTheme.name = "qtct";
};
```

2. check for styles in qt6ct

### Expected behavior

the qt styles (tested: kvantum or breeze) should show up

### Additional context
the styles `windows` & `fusion` are still available
the styles are discoverable in https://github.com/NixOS/nixpkgs/commit/3f16bb4a40a55753cb84be9a54cd42c16222de6e

not in https://github.com/NixOS/nixpkgs/commit/b857c509dff9e4975abedca573a67b1bdd0c557c
not in https://github.com/NixOS/nixpkgs/commit/14041545954d8860af8b90b197261d34cefe9014

### Notify maintainers

@Flakebi @Scrumplex

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.57, NixOS, 24.11 (Vicuna), 24.11.20241018.4c2fcb0`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Lix, like Nix) 2.91.0
 - nixpkgs: `/nix/store/riqkpszjqk02bi1wppfg8ip5xvh102qd-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by DarkKronicle** [comment]
Looked into this, and found what I think is the issue:

https://bugreports.qt.io/browse/QTBUG-130109

**Comment by Erothas** [comment]
I have been scratching the heck out of my head with this exact issue for days by now. I have recently switched from Hyprland to KDE. I did not experience any issues whatsoever until I went through with that migration, and consequently believed KDE (and NixOS) to be the culprit behind it all. Apparently it's not, and that there is a planned fix already on the way, judging from @DarkKronicle's links.

Good to know. Thank you for the heads up!
